app/controllers/userController.js

```diff
@@ -12,25 +12,27 @@ exports.verifyID = function (req, res) {
             'password': password
         }
     }, function (error, response, body) {
+        if (error) return console.error(error);
```

Comment:
I don't know how to make this userController better in view and logic, please help me refactor it if possible, THX🙏